src/BookData.js

```diff
@@ -1,4 +1,4 @@
-const book = [
+const books = [
```

Comment:
Food for thought. This would be a good opportunity for a `json` file. If you're not familiar, they're basically just data files that look almost exactly like this, but don't get run by the Javascript interpreter. 
Something like `books.json` would be good here! And then the contents could just be:

```
[
  { id: 1, author: 'Test Author', title: 'Test Title', genre: 'Test Genre' },
  { id: 2, author: 'Test Author', title: 'Test Title', genre: 'Test Genre' },
  { id: 3, author: 'Test Author', title: 'Test Title', genre: 'Test Genre' },
  { id: 4, author: 'Test Author', title: 'Test Title', genre: 'Test Genre' },
  { id: 5, author: 'Test Author', title: 'Test Title', genre: 'Test Genre' },
  { id: 6, author: 'Test Author', title: 'Test Title', genre: 'Test Genre' },
  { id: 7, author: 'Test Author', title: 'Test Title', genre: 'Test Genre' },
  { id: 8, author: 'Test Author', title: 'Test Title', genre: 'Test Genre' },
  { id: 9, author: 'Test Author', title: 'Test Title', genre: 'Test Genre' },
  { id: 10, author: 'Test Author', title: 'Test Title', genre: 'Test Genre' },
  { id: 11, author: 'Test Author', title: 'Test Title', genre: 'Test Genre' },
  { id: 12, author: 'Test Author', title: 'Test Title', genre: 'Test Genre' },
]```

Reply:
So I definitely thought about this, but didn’t know how to implement it. In order to access it do I just import books.json to whatever file I’m using it in?